gazebo_ros2_control_demos/urdf/test_gripper_mimic_joint.xacro.urdf

```diff
@@ -71,7 +71,12 @@
       <plugin>gazebo_ros2_control/GazeboSystem</plugin>
     </hardware>
     <joint name="right_finger_joint">
-      <command_interface name="position"/>
+      <xacro:if value="$(arg effort)">
```

Comment:
I propose here to add a completelly separate file for the effort example.
It is example code, so we should try to make it as clean and simple as possible, even if we have to duplicate some parts.

What do you think?

Reply:
I'm a friend of not duplicating code, but I do understand your input for example code. I'll change that.